manual-provision-payg/config.json

```diff
@@ -1,5 +1,4 @@
 {
-"InstructionsLine1": "To get a new unlock code send the amount required to Caletre, with the account number as $device_id.",
-"InstructionsLine2": "You will then receive an unlock code by SMS. Use this code to unlock time on your computer."
-
+"InstructionsLine1": "To get a new unlock keycode please contact your vendor with the account number as $device_id.",
+"InstructionsLine2": "You will then receive a keycode by SMS to unlock and add usage time to your computer."
```

Comment:
We'll probably need to translate these, depending on the location of the field trials :)